CVChatbot/CVChatbot.Bot/ChatbotActions/Commands/Stats.cs

```diff
@@ -57,7 +57,7 @@ public override ActionPermissionLevel GetPermissionLevel()
 
         protected override string GetRegexMatchingPattern()
         {
-            return "^(close vote )?stats( (please|plz))?$";
+            return "^(close vote )?stats( (please|pl[sz]))?$";
```

Comment:
I switched my "please" pattern half way through. We should make it consistence throughout.